src/components/walletconnect/ProposalForm/index.tsx

```diff
@@ -37,14 +37,16 @@ const ProposalForm = ({ proposal, onApprove, onReject }: ProposalFormProps): Rea
         WalletConnect
       </Typography>
 
-      <div className={css.icon}>
-        <SafeAppIconCard
-          src={proposer.metadata.icons[0]}
-          width={32}
-          height={32}
-          alt={`${proposer.metadata.name} logo`}
-        />
-      </div>
+      {proposer.metadata.icons[0] && (
+        <div className={css.icon}>
+          <SafeAppIconCard
+            src={proposer.metadata.icons[0]}
+            width={32}
+            height={32}
+            alt={`${proposer.metadata.name} logo`}
```

Comment:
In this instance it cannot be empty?

Reply:
Addressed in cc9fb09a2cf9eab471f4ad70c4fcac55acde0264.